pip/commands/install.py

```diff
@@ -201,6 +201,7 @@ def _build_package_finder(self, options, index_urls, session):
         )
 
     def run(self, options, args):
+        logger.debug('%s: In %s.run', __file__, self.__class__.__name__)
```

Comment:
Yeah. This is a pattern I use when I feel like I'm going to add a bunch of logging. I can copy and paste that line.